pkg/certificate/providers/vault/tools.go

```diff
@@ -11,12 +11,8 @@ func getDurationInMinutes(validityPeriod time.Duration) string {
 	return fmt.Sprintf("%dh", validityPeriod/time.Hour)
 }
 
-func getIssueURL(role vaultRole) vaultPath {
-	return vaultPath(fmt.Sprintf("pki/issue/%+v", role))
-}
-
-func getRoleConfigURL(role vaultRole) vaultPath {
-	return vaultPath(fmt.Sprintf("pki/roles/%s", role))
+func getroleURL(role string) string {
```

Comment:
Did you intend to make change?  The functionality should stay the same (as well. as the func names), with simply changing the type from. vaultRole to string

Reply:
🤦‍♀️ went too hard on the cleanup and thought "getIssueURL" and "getRoleConfigURL" were the same thing. my b!